assets/js/account/account-orders.tpl.liquid

```diff
@@ -17,16 +17,16 @@
             <table class="table table-striped border-top-light sortable orders-list-table">
                 <thead>
                     <tr>
-                        <th ng-click="$ctrl.sortChanged('number')" ng-class="{'desc': $ctrl.getSortDirection('number') == $ctrl.sortDescending, 'asc': $ctrl.getSortDirection('number') == $ctrl.sortAscending }">{{ 'customer.orders.order_number' | t }}</th>
+                        <th ng-click="$ctrl.sortChanged('number')" class="col-sortable" ng-class="{'desc': $ctrl.getSortDirection('number') == $ctrl.sortDescending, 'asc': $ctrl.getSortDirection('number') == $ctrl.sortAscending }" class="">{{ 'customer.orders.order_number' | t }}</th>
```

Comment:
remove this second class
![image](https://user-images.githubusercontent.com/22875828/93072979-7ce84680-f682-11ea-9fac-01ae255c4f76.png)